README.md

```diff
@@ -146,7 +146,37 @@ data directory.
 `tezos-{accuser, endorser}` have configurable node address, so that they can be used with both
 remote and local node.
 
-### Systemd units on other systems
+### Launchd services on macOS
+
+`tezos-accuser-<proto>`, `tezos-baker-<proto>`, `tezos-endorser-<proto>` formulas
+provide backround services for running corresponding daemons.
+
+Since `tezos-node` and `tezos-signer` provide multiple services they are provided
+in a dedicated meta-formulas. These formulas doesn't install any binaries and only provide
+backround services.
+
+Formulas with `tezos-node` background services:
+* `tezos-node-mainnet`
+* `tezos-node-edo2net`
+
+Formulas with `tezos-signer` background services:
+* `tezos-signer-http`
+* `tezos-signer-https`
+* `tesos-signer-tcp`
+* `tezos-signer-unix`
+
+To start the service: `brew services start <formula>`.
+
+To stop the services: `brew services stop <formula>`.
+
+All of the brew services have various configurable env variables. These variables
+can be changed in the corresponding `usr/local/Cellar/tezos-signer-tcp/<version>/homebrew.mxcl.<formula>.plist`.
```

Comment:
Is there no easier way to modify these files, e.g. a launchd `edit` option?

Reply:
I suppose, there is a GUI tool, but I can't check, because I don't have mac with GUI :cry:

Reply:
So, AFAIU, there is no anything related to `edit` in both `launchctl` and `brew services` and the suggested way is to use any XML editor (or just any text editor)